### ISSUE TITLE
Support for continuous output?

### ISSUE THREAD

**Issue by ma-sadeghi** [qubvel/segmentation_models.pytorch#651]
Thank you for creating this great package! My question might not be very closely related to image segmentation, but I'll ask anyway: is it possible to use your package for continuous output data? My input data are binary and the output are continuous (floats between 0-1). So, it's in a way opposite the typical image segmentation. Thank you so much!

**Comment by JulienMaille** [comment]
Just get rid of the activation layer?

**Comment by Geethen** [comment]
I have the same question

I have currently setup the arch like so

model = smp.Unet(
    encoder_name="resnet18",   # choose encoder, e.g. mobilenet_v2 or efficientnet-b7
    encoder_weights= None,     # use `imagenet` pre-trained weights for encoder initialization
    in_channels= 4,           # model input channels (1 for gray-scale images, 3 for RGB, etc.)
    classes = 1,              # model output channels (number of classes in your dataset)
    activation = 'identity' 
)

when I print the model. I see that all the decoder activations are 'identity' not just the final activation (which is needed for regression).

 Any suggestions?

**Comment by ma-sadeghi** [comment]
@JulienMaille Sorry for the super late reply. I actually used "sigmoid" for the output layer and it worked as I wanted. Thanks!
